ee/debug/checkups/checkpoint.go

```diff
@@ -95,12 +128,16 @@ func (c *logCheckPointer) Once(ctx context.Context) {
 
 	// Compute score from warning, passing, and failing counts
 	scoredCheckups := warningCount + failingCount + passingCount
-	score := (float64(passingCount+(warningCount/2)) / float64(scoredCheckups)) * 100
+	score := ((passingCount + (warningCount / 2)) / scoredCheckups) * 100
 	observability.CheckupScoreGauge.Record(ctx, score)
 
 	c.slogger.Log(ctx, slog.LevelDebug,
 		"computed checkup score",
 		"score", score,
+		"failing_count", failingCount,
```

Comment:
thinking about how we're going to hone to this logic (and any thresholds we're working on)- I see we're recording the metric but do we have a quick way to get an idea of why devices have that score? I'm wondering if we should at least start with this log being Info level, and including the names for failing and passing checkups. Just so we can log search on this and see any trends worth digging into, rather than investigating one device at a time based on score

Reply:
How about `Warn` level if the score isn't 100%, `Debug` (or `Info`) level otherwise, and then adding the names of the checkups?

Reply:
works for me!

Reply:
Updated